template-deliverables.md

```diff
@@ -0,0 +1,145 @@
+
+A: Yes, but try not to go much deeper than level 3.
+
+
+## Template guide
+
+Example filename: abc-template-guide.md
+
+This guide provides a deeper explanation of how to fill in the raw template.
+It provides a lightweight introduction to the purpose of this documentation and explains how to fill in each section of the document.
+
+In essence, this guide supports the key decisions that the template user might need to make during the writing process.
+
+This guide might also optionally include a checklist of the required components for this type of document.
+It might also include a list of helpful resources or examples that were consulted when the template was created.
+
+
+### Content and formatting guidelines
+
+The template guide should follow the same basic structure:
+
+- **Template guide title** - The type of document this template is for and the words "guide." For example, "Tutorial guide" or "README guide." This title should be formatted with the heading 1 weight.
+- **Introductory comment** - Include an embedded writing tip at the start that tells users they should first read this guide and the before-you-start guide before they fill in the template. This comment could also mention how to get the raw template file. Put introductory comments in {curly brackets}.
+- **Why do I need this type of document?** - Provide some of the information about the purpose of this type of document and how it helps your documentation project. The header for this section should be formatted with the heading 2 weight.
+- **Content of this template** - This header marks the beginning of the part of the guide that explains how to fill in each section of the guide. It should be formatted with the heading 2 weight.
+- **About the {insert section name here} section** - Next, include each section as they appear in the template with the heading 3 weight. For example, if your template includes a "Before you start" section, this heading should say "About the before your start section." And then it should provide guidance about what kind of content goes in that section and why that section might be included in the final document. If the section is optional, indicate why some documents could benefit from that section or why it might be left out. If the template user needs to make a decision about the content in that section, provide guidance about what should go into that section.
+- **Checklist (optional)** - If it makes sense to include a checklist of the typical content required for this type of document, include that here.
+- **Additional resources (optional)** - If you’d like to point the user to any helpful guides for deeper study or if you want to acknowledge any resources that inspired you while you wrote this template, link to or cite those resources here.
+
+### Frequently asked questions
+
+Q: No questions yet.
+
+
+## Before you start guide (optional)
+
+Example filename: before-you-start-abc-template.md

```

Comment:
I recommend we stick the convention of having the template name at the beginning of each file, and the words that distinguish this file from others in the template set at the end. 

So that would mean this should be something like: abc-before-you-start.md

Reply:
This is where "primer" might be better.

Reply:
I've adopted this suggestion.